superset-frontend/src/features/databases/UploadDataModel/UploadDataModal.test.tsx

```diff
@@ -68,6 +69,13 @@ const excelProps = {
   type: 'excel',
 };
 
+const columnarProps = {
+  show: true,
+  onHide: () => {},
+  allowedExtensions: ['parquet', 'zip'],
```

Comment:
It seems weird to me that we associate `zip` with Parquet files. What if I want to upload a zipped CSV or Excel file?

I mentioned this before, but ideally we'd have a single endpoint that would analyze the file (unzipping it if needed) and send it to the appropriate uploader.